File: src/org/elixir_lang/psi/call/name/Quote.java
package org.elixir_lang.psi.call.name;

import static org.elixir_lang.psi.call.name.Module.KERNEL;

public class Quote {
    public static final String FUNCTION = "quote";
    public static final String MODULE = KERNEL;
    public static class KeywordArgument {
        public static final String LOCATION = "location";
        public static final String UNQUOTE = "unquote";
    }
}

File: src/org/elixir_lang/semantic/textual/Quote.kt
package org.elixir_lang.semantic.textual

import com.intellij.psi.ElementDescriptionLocation
import com.intellij.psi.PsiElement
import org.elixir_lang.psi.Quote
import org.elixir_lang.semantic.Textual

class Quote(val quote: Quote) : Textual {
    override val psiElement: PsiElement
        get() = quote

    override fun elementDescription(location: ElementDescriptionLocation): String? {
        TODO("Not yet implemented")
    }
}
